bip32/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "bip32"
-version = "0.5.0"
+version = "0.5.1"
```

Comment:
Please don't bump the version number in a feature PR. We have a release process that includes updating the CHANGELOG.

Reply:
Sorry, I'm used to crates asking me to bump it :), will revert